source/Halibut/Transport/Streams/NetworkTimeoutStream.cs

```diff
@@ -114,7 +115,8 @@ void ThrowMeaningfulException(Exception? innerException = null)
             {
                 if (timeoutCancellationTokenSource.IsCancellationRequested)
                 {
-                    throw new OperationCanceledException($"The {methodName} operation timed out after {TimeSpan.FromMilliseconds(timeout)}.", innerException);
+                    var socketException = new SocketException(10060);
```

Comment:
This magic number is what is thrown when the synchronous version is thrown. It also generates the same message for us 😁